README.md

```diff
@@ -1,20 +1,54 @@
 # SpeedBagIt
 [![Build Status](https://travis-ci.org/NCEAS/eml.svg?branch=master)](https://travis-ci.org/NCEAS/eml)
```

Comment:
The `/NCEAS/eml.svg` links in the Build status badge are incorrect and should be `/DataONEorg/speedbagit.svg`. Probably also best to use travis-ci.com rather than .org, as org is shutting down next month.